# Tomato gene explorer 

A Shiny application to explore RNA-seq results from processed (= gene counts) data coming from mRNA-seq data. 

# Data provenance (ITAG2.4)

## Genome references

Genomic references can be found on Zenodo [Link](https://zenodo.org/record/5761962). 

Genome assembly and annotation ITAG2.4 were used:  
- Genome FASTA: [S_lycopersicum_chromosomes.2.50.fa.gz](https://zenodo.org/api/files/c5778399-5188-4959-93b7-1738584c0f72/S_lycopersicum_chromosomes.2.50.fa.gz)     
- Genome annotation: [ITAG2.4_gene_models.gtf](https://zenodo.org/api/files/3e34c90f-9fce-4947-9ad3-0573572d942b/ITAG2.4_gene_models.gtf)  

## Snakemake pipeline 

To get the scaled gene counts, the v0.3.8 release of the __snakemake_rnaseq__ pipeline was used:  
- [Link GitHub](https://github.com/BleekerLab/snakemake_rnaseq/releases/tag/v0.3.8)  
- [Link Zenodo](https://doi.org/10.5281/zenodo.5547867)  


## Dataset 01: 20 cultivated and wild tomato accessions

mRNA-seq files (5 parts):  
- [Part 1](https://zenodo.org/record/3603295)  
- [Part 2](https://zenodo.org/record/3610352)  
- [Part 3](https://zenodo.org/record/3610997)  
- [Part 4](https://zenodo.org/record/3611053)  
- [Part 5](https://zenodo.org/record/3611144)  

## Dataset 02: cultivated and wild tomato tissues

mRNA-seq files: [Zenodo link](https://zenodo.org/record/4326969)

Samples S31 to S48 were used. 


## Dataset 03: S. lyc Elite x S. hab PI127826 F2 lines

mRNA-seq files (two parts):  
- Parents and F1 (2017): [Zenodo link](https://zenodo.org/record/4322913)  
- F2 (2019): [Zenodo link](https://zenodo.org/record/4491747)  

## Dataset 04: MYC1 knock-out and knock-down mutants

mRNA-seq files [Zenodo link](https://zenodo.org/record/4706910)
