# About

## The tomato gene explorer
RNA-seq datasets were aligned against the tomato genome (SL2.50 assembly) together with the annotation ITAG2.4 using the [snakemake-rnaseq pipeline release v0.3.8](https://github.com/BleekerLab/snakemake_rnaseq/releases/tag/v0.3.8).

The tomato genome references can be found on SOl Genomics:
- [Genome assembly](https://solgenomics.net/ftp/tomato_genome/Heinz1706/assembly/build_2.50/).   
- [Genome annotation](https://solgenomics.net/ftp/tomato_genome/Heinz1706/annotation/ITAG2.4_release/).  


## Datasets 
These RNA-seq datasets are archived at Zenodo and, for some of them, at the EBI European Nucleotide Archive with accession numbers. 

1. Tomato active and lazy trichomes: 
  - [2017 Zenodo data record (parents, F1)](10.5281/zenodo.3603229) 
  - [2019 Zenodo data record (parents, F2)](https://doi.org/10.5281/zenodo.3610278). 
2. Tomato tissues: bald stems, leaf and leaf primordia. [Zenodo 2020 data record](10.5281/zenodo.3954272).  
3. Cultivated and wild tomato stem trichomes (2016): ENA accession **PRJEB44371**.
4. MYC1-related genotypes (2020): ENA accession **PRJEB49287**.

## Credits

Marc Galland, support data analyst, Plant Physiology Department, Green Life Sciences, SILS-FNWI, University of Amsterdam. 

## Useful links
- [Plant Physiology Department](https://sils.uva.nl/content/research-groups/plant-physiology/plant-physiology.html).    
- [Green Life Sciences cluster website](https://sils.uva.nl/research/collaboration-initiatives/green-life-sciences/green-life-sciences.html).  
- Find other local made Shiny apps on the [Science Park Study Group tools page](https://scienceparkstudygroup.info/tools/).  

